tests/test_core.py

```diff
@@ -1107,6 +1107,8 @@ def test_uiedgeinsets_repr(self):
     def test_duplicate_class_registration(self):
         """If you define a class name twice in the same runtime, you get an error."""
 
+        ObjCClass.auto_rename = False
```

Comment:
I suspect this is only needed because `test_class_auto_rename` is setting the property; if that class-based property has an impact on other tests, then the test that sets the property should clean up and unset it.